src/Sulu/Bundle/DocumentManagerBundle/DependencyInjection/Configuration.php

```diff
@@ -72,7 +72,7 @@ public function getConfigTreeBuilder()
                 ->end()
                 ->scalarNode('slugifier')
                     ->defaultValue('Sulu\Bundle\DocumentManagerBundle\Slugifier\Urlizer::urlize')
-                    ->setDeprecated()
+                    ->setDeprecated('The "sulu_document_manager.slugifier" is deprecated since 2.1. To be removed 3.0.')
```

Comment:
The "not used" part was very important to me, otherwise people will still wonder if this does not have any effect.